### PR TITLE
[DX-913] Add release notes for the portal v1.8.2

### PR DESCRIPTION
**Pull request by slega**
### For internal users - Please add a Jira DX PR ticket to the subject!
https://tyktech.atlassian.net/browse/DX-913
<br>

---

### Preview Link
https://deploy-preview-3792--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2/
<br>

### Description
Add release notes for EDP v1.8.2
<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+# Release Highlights
+The 1.8.2 release addresses multiple high-priority bugs:
+- Fixed the bug where an API Consumer could add incompatible products to a cart making its state inconsistent.
+- Fixed the bug where it was possible to use the same session cookie from different IP addresses making the portal vulnerable to the [Cross-Site Request Forgery (CSRF) attack](https://en.wikipedia.org/wiki/Cross-site_request_forgery).
+- Fixed the bug where an admin user couldn't create a new team in Kubernetes environment.
+- Fixed the bug where the navigation in the API documentation was broken when Redoc is selected as a documentation rendering engine.
+
+# Download
+- [Docker image to pull](https://hub.docker.com/layers/tykio/portal/v1.8.0/images/sha256-d93fcfbbcc4a72d3f6abf49ce65f234e6e65915a43cca3a30d5376e5fab2d644?context=explore)
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md]: Remember to update docker link

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- [The default theme package](https://github.com/TykTechnologies/portal-themes/blob/main/v1.8.0/default.zip)
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md]: Remember to update source link for 1.8.2 theme

**Review by dcs3spp — COMMENTED** [review]
PR looks good to me @slega 

Docker and source links will need to be updated, before this is released

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md (hunk 2 above)
Comment (slega) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md]: Done

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md (hunk 1 above)
Comment (slega) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.8.2.md]: Done

**Review by dcs3spp — APPROVED** [review]
pr lgtm

**Pull request merged**